.github/workflows/codeql-analysis.yml

```diff
@@ -30,9 +29,23 @@ jobs:
       - name: Initialize CodeQL
         uses: github/codeql-action/init@v1
         with:
-          languages: csharp
+          languages: csharp, python
 
-      - name: Install .NET SDK
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ];
+          then pip install -r requirements.txt;
+          fi
+          # Set the `CODEQL-PYTHON` environment variable to the Python executable
+          # that includes the dependencies
+          echo "CODEQL_PYTHON=$(which python)" >> $GITHUB_ENV
```

Comment:
Can you check to see if I did this right, for CodeQL?  Just trying to add Python here to security scanning.

Reply:
It looks like it's running okay, but wanted a second opinion.

Reply:
I don't think it's working as you originally intended. You are installing Python dependencies manually. 
The problems are:
1. We don't have `requirements.txt`.(But, I think it's better to make this file.)
2. Needs to set `setup-python-dependencies: false`

You can check official docs here: [Analyzing Python dependencies](https://docs.github.com/en/code-security/secure-coding/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#analyzing-python-dependencies).

Reply:
Yah, I copied code from there but missed the 2nd point.  I've also removed the `requirements.txt` option since we're not using it.  I think this is working now.